### ISSUE TITLE
Add clarification about v1 support and v2 release timeline

### ISSUE THREAD

**Issue by petewall** [grafana/k8s-monitoring-helm#809]
If people are seeing this now:

* v1 will keep support for the time-being. No planned time for it to be deprecated.
   * It will continue to get bugfixes and minor updates, no new major features likely.
* v2's initial release is planned for mid-November.
* Support for both will run in parallel for a while, but v1 will be deprecated at some point.

**Comment by nscherer-chipotle** [comment]
Hello!

Could you confirm if the v1 chart will keep getting new versions of the Alloy agent as they are released? I.E. can we stay on the V1 chart and still meet normal patching requirements? (Until the V1 chart is deprecated, obviously.)

Thanks!

**Comment by petewall** [comment]
Yep! Helm chart dependencies will continue to be applied to the v1 chart until support is officially removed.

**Comment by petewall** [comment]
i mean... v2 is released... closing.
